logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/CallRepository.kt

```diff
@@ -359,6 +364,22 @@ internal class CallDataSource(
         }
     }
 
+    override fun updateIsCbrEnabled(isCbrEnabled: Boolean) {
+        val callMetadataProfile = _callMetadataProfile.value
+        val conversationId = getEstablishedCall().conversationId
+        callMetadataProfile.data[conversationId]?.let { callMetadata ->
+            val updatedCallMetaData = callMetadataProfile.data.toMutableMap().apply {
+                this[conversationId] = callMetadata.copy(
+                    isCbrEnabled = isCbrEnabled
+                )
+            }
+
+            _callMetadataProfile.value = callMetadataProfile.copy(
+                data = updatedCallMetaData
+            )
+        }
+    }
+
```

Comment:
Just asking to make sure, can `getEstablishedCall()` ever return null? Or as on upon receiving a CBR value change it will always be when a call is established?

Reply:
yeah, `updateIsCbrEnabled` is called in `constantBitRateStateChangeHandler` that will happens only when there is an active call